moveit_core/utils/include/moveit/utils/rclcpp_utils.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
```

Comment:
Please add license headers. Are there currently efforts towards porting this?

Reply:
I'll port it to the ROS2 repo